index.html

```diff
@@ -293,7 +293,7 @@ <h3>
           </p>
 
           <pre class="example nohighlight" title="Creating the DID with optional path and port">
-did:web:example.com%3A3000:user:alice 
+did:web:example.com%3A3000:user:alice
```

Comment:
Not going to argue in this PR, but I strongly feel that we should shift to this:

```
did:web:example.com/user/alice.json
```

Reply:
+1

I can live with that.

If we're going to simplify, we might as well take it all the way.

I have a roadmap item to support condensed representations at some point next year, but I can cross-reference `.json` via `did:web` with [Also Known As](https://www.w3.org/TR/did-core/#also-known-as) for compatibility.